types/cesium/index.d.ts

```diff
@@ -2392,7 +2392,7 @@ declare namespace Cesium {
         outlineWidth: Property;
         perPositionHeight: Property;
         constructor(options?: {
-            hierarchy?: Property;
+            hierarchy?: Property | Cartesian3[];
```

Comment:
Based on [this](https://cesiumjs.org/Cesium/Build/Documentation/PolygonGraphics.html?classFilter=polygongra) documentation should't this be `PolygonHierarchy` instead?